chain/index/indexer.go

```diff
@@ -54,8 +65,8 @@ type SqliteIndexer struct {
 	db *sql.DB
 	cs ChainStore
 
-	idToRobustAddrFunc       IdToRobustAddrFunc
-	recomputeTipSetStateFunc recomputeTipSetStateFunc
+	idToRobustAddrFunc IdToRobustAddrFunc
+	eventLoaderFunc    eventLoaderFunc
```

Comment:
This is slightly misnamed, maybe it should be `executedMessagesLoaderFunc` and modify references to it wherever it's used, including public methods. You also have `MakeLoadExecutedMessages` in here to make one of these, so that's already close, but has "load" in it. Need to align the naming a bit better across all public methods I think, and having internal references match would be helpful too.

Reply:
Done.